src/App.js

```diff
@@ -1,18 +1,31 @@
 
 import './App.css';
-import Articles from "./components/Articles"
+import Topics from "./components/Topics"
+import HomePage from "./components/Homepage"
+import {useState} from "react"
+import {BrowserRouter, Link, Route, Routes, useParams} from "react-router-dom"
 
 
 	
 
 function App() {
+
+const [topic, setTopic] = useState("")
+
   return (
+    <BrowserRouter>
     <div className="span-s1">
       <header className="App-header">
-        Alex's News APP
+        <Link to="/">Alex's News APP</Link>
       </header>
-       <Articles /> 
-    </div>
+      </div>
+      <Routes>
+        <Route path="/" element={<Topics topic={topic} setTopic={setTopic}/>}/>
```

Comment:
Not sure why you need topic and setTopic. Seems like they haven't been using anywhere else